test/unit/middlewares/user-validator.test.js

```diff
@@ -128,5 +149,49 @@ describe("Middleware | Validators | User", function () {
       });
       expect(nextSpy.calledOnce).to.be.equal(false);
     });
+
+    it("Stops the propagation of the next if linkedin id is invalid", async function () {
+      const req = {
+        body: {
+          last_name: "patil",
+          first_name: "Abhay",
+          username: "invalidusername12",
+          twitter_id: "abhayisawesome",
+          linkedin_id: "@abhay2011",
+        },
+      };
+      const res = {
+        boom: {
+          badRequest: () => {},
+        },
+      };
+      const nextSpy = sinon.spy();
+      await updateUser(req, res, nextSpy).catch((err) => {
+        expect(err).to.be.an.instanceOf(Error);
+      });
+      expect(nextSpy.calledOnce).to.be.equal(false);
+    });
+
+    it("Stops the propagation of the next if linkedin id is invalid or contain spaces", async function () {
+      const req = {
+        body: {
+          last_name: "patil",
+          first_name: "Abhay",
+          username: "invalidusername12",
+          twitter_id: "abhayisawesome",
+          linkedin_id: "abhay 2011",
```

Comment:
@mailmesriza98 Similar feature we are working on currently is that if the username is the same it will append it with a number, and this will happen before the signup itself.